### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by peanutbother**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,5 +4,9 @@
   <title>comroid.de</title>
 </head>
 <body>
-  <iframe src="https://discordapp.com/widget?id=297846944754040842&theme=dark" width="350" height="500" allowtransparency="true" frameborder="0"></iframe>
```
Comment (burdoto) [inline index.html]: Please try to implement this embed somehow, doesn't have to be nice, but at least it should be visible. The embed is already configured.

Comment (peanutbother) [reply index.html]: There's this href for. 
We can't iframe discord.

Comment (burdoto) [reply index.html]: Tho this iframe is a 1:1 template issued by discord

Comment (peanutbother) [reply index.html]: This might be true for the main website but does this apply to the invite site too?

Comment (burdoto) [reply index.html]: No. That one shouldn't be embedded. This was the already changed embed 😜

**Pull request merged**